components/tools/OmeroWeb/omeroweb/webclient/static/webclient/javascript/ome.tree.js

```diff
@@ -390,8 +390,11 @@ $(function() {
             'data' : function(node, callback, payload) {
                 // Get the data for this query
                 if (payload === undefined) {
-                    payload = {};
+                    payload = this.element.data('payload');
```

Comment:
I think a bit more info could be handy to remind us (me) or the use-case here, E.g:
// Check for existing 'payload' data, used to initialise the jsTree